rectools/models/rank/compat.py

```diff
@@ -0,0 +1,7 @@
+from rectools.compat import RequirementUnavailable
+
+
+class TorchRanker(RequirementUnavailable):
```

Comment:
Could you add this class [here](https://github.com/MobileTeleSystems/RecTools/blob/main/tests/test_compat.py#L39) for test?  
Then the whole file will be covered

Reply:
done